Solved-Problems/Backspace_String_Compare/Main.java

```diff
@@ -1,3 +1,12 @@
+
+//Problem Name: 844.Backspace String Compare
+
+//Approach: Using Stacks
+
+//Difficulty: Easy
+
+//Link: https://leetcode.com/problems/backspace-string-compare/
```

Comment:
I believe it's a simple misunderstanding, there's no gap between the comments. Please remove those unnecessary new lines.

```suggestion
//Problem Name: 844.Backspace String Compare
//Approach: Using Stacks
//Difficulty: Easy
//Link: https://leetcode.com/problems/backspace-string-compare/
```